spec/reports/cost_report_spec.rb

```diff
@@ -91,28 +85,43 @@ def add_cluster_htitem(*htitems)
 
   describe "#cost_per_volume" do
     it "calculates cost per volume" do
-      expect(cr.cost_per_volume).to eq(cr.target_cost / cr.num_volumes)
+      # 3 volumes, target cost $10
```

Comment:
I found it hard to understand what the state of the database was given the various before blocks and additional stuff being added. I figured being explicit in each test about what's there was the best option.